examples/misc/upload_media_bundle.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example uploads an HTML5 zip file as a media bundle."""
+
+
+import argparse
+import sys
+import requests
```

Comment:
*Nitpick*: order of imports should be:

```
import argparse
import requests
import sys
```

Reply:
fixed

Reply:
Actually this now generates a gpylinter complaint that it should be:
```
import argparse
import sys

import requests
```

Reply:
Whoops, your linter is correct on this one. What I said would have been correct if those were all standard library imports, but requests is not. You should bundle requests with other installed library imports.